src/components/Result/ResultContent/TextResult.tsx

```diff
@@ -17,26 +17,37 @@ const TextResult = ({ textSummaryResponse }: TextResultPropType) => {
           </div>
         </div>
         <p className="my-8 text-base leading-6 md:leading-8">{textSummaryResponse?.summary}</p>
-        <div className="block md:hidden">
-          <CopyButton textToCopy={textSummaryResponse?.summary} />
-        </div>
-        <div className="mt-16 flex items-center justify-between">
-          <h3 className="text-xl font-bold">Key Points & Analysis</h3>
-          <div className="hidden md:block">
-            <CopyButton textToCopy={textSummaryResponse?.keyPoints?.join("\n")} />
-          </div>
-        </div>
-        <ul className="my-8 list-disc px-6 text-base leading-6 md:leading-8">
-          {textSummaryResponse.keyPoints?.map((keyPoint, id) => (
-            <li key={id}>{keyPoint}</li>
-          ))}
-        </ul>
-        <div className="mb-4 text-base leading-6 md:leading-8">
-          This articles has a {textSummaryResponse.bias} bias and {textSummaryResponse.tone} tone
-        </div>
-        <div className="block md:hidden">
-          <CopyButton textToCopy={textSummaryResponse?.keyPoints?.join("\n")} />
-        </div>
+        {textSummaryResponse?.keyPoints?.length > 0 && (
+          <>
+            <div className="block md:hidden">
+              <CopyButton textToCopy={textSummaryResponse?.summary} />
+            </div>
+            <div className="mt-16 flex items-center justify-between">
+              <h3 className="text-xl font-bold">Key Points & Analysis</h3>
+              <div className="hidden md:block">
+                <CopyButton
+                  textToCopy={textSummaryResponse?.keyPoints?.join("\n")}
+                />
+              </div>
+            </div>
+            <ul className="my-8 list-disc px-6 text-base leading-6 md:leading-8">
+              {textSummaryResponse?.keyPoints?.map((keyPoint, id) => (
+                <li key={id}>{keyPoint}</li>
+              ))}
+            </ul>
+          </>
+        )}
+
+        {textSummaryResponse?.bias && textSummaryResponse?.tone && (
+          <>
+            <div className="mb-4 text-base leading-6 md:leading-8">
+              This article has a {textSummaryResponse?.bias} bias and {textSummaryResponse?.tone} tone
```

Comment:
maybe this should just be:
`This content has a`

because it's not necessarily an article

Reply:
@MaiaS good catch!

Reply:
May be we should generate this line from chatGPT too? because the content bias could start with a vowel too